frontend/src/pages/auth/components/fields/email/EmailField.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {useEffect, useState} from 'react';
+import {ValidationErrorMessages} from "../../../../../common/constansts/validationErrorMessages";
+import {TextField} from "@mui/material";
+import {EmailFieldProps} from "./props";
+import {EMAIL_REGEX} from "../../../../../common/constansts/validation";
+
+const EmailField: React.FC<EmailFieldProps> = ({
+                                                   defaultEmail,
+                                                   updateEmailFields
+                                               }) => {
```

Comment:
I would do it inline, as there's enough space 
`const EmailField: React.FC<EmailUserCredentialsProps> = ({defaultEmail, updateEmailFields}) => {`